compose/service.py

```diff
@@ -937,7 +938,19 @@ def build_volume_binding(volume_spec):
 
 
 def parse_volume_spec(volume_config):
+    """
+    A volume_config string, which is a path, split it into external:internal[:mode]
+    parts to be returned as a valid VolumeSpec tuple.
+    """
     parts = volume_config.split(':')
+
+    if IS_WINDOWS_PLATFORM:
+        # relative paths in windows expand to include the drive, eg C:\
+        # so we join the first 2 parts back together to count as one
+        windows_parts = [":".join(parts[0:2])]
+        windows_parts.extend(parts[2:])
+        parts = windows_parts
```

Comment:
Could we do this as part of the config module instead?  If we have config always return a proper `VolumeSpec()` we don't have to worry about it twice